tests/microgrid/power_distributing/_component_status/test_ev_charger_status.py

```diff
@@ -121,9 +132,12 @@ async def test_status_changes(self, mocker: MockerFixture) -> None:
                     _EV_CHARGER_ID,
                     datetime.now(tz=timezone.utc),
                     active_power=0.0,
-                    component_state=EVChargerComponentState.READY,
-                    cable_state=EVChargerCableState.EV_LOCKED,
-                )
+                    states={
+                        ComponentStateCode.READY,
+                        ComponentStateCode.EV_CHARGING_CABLE_PLUGGED_AT_EV,
+                        ComponentStateCode.EV_CHARGING_CABLE_LOCKED_AT_STATION,
+                    },
```

Comment:
this is not an exact match.

Old says EV LOCKED.  new says EV PLUGGED.

Reply:
Oh, good catch! Will fix.

Reply:
The test still passed because actually all permutations are considered as *connected* I guess.

Reply:
(fixed anyway)